test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/FormFileModelBinderTest.cs

```diff
@@ -82,17 +82,15 @@ public class FormFileModelBinderTest
 
             // Assert
             Assert.NotEqual(ModelBindingResult.NoResult, result);
-            var files = Assert.IsAssignableFrom<IList<IFormFile>>(result.Model);
-            Assert.Equal(2, files.Count);
+            var file = Assert.IsAssignableFrom<IFormFile>(result.Model);
+            Assert.Equal("file1.txt", file.FileName);
         }
 
         [Fact]
-        public async Task FormFileModelBinder_ExpectSingleFile_BindFirstFile()
+        public async Task FormFileModelBinder_ReturnsNoResult_WhenNoFilePosted()
```

Comment:
It's a problem with the diff actually, if you run diff --patience it will show the right diff. I don't know how to make GitHub do that though. I will show you in person